src/index.html

```diff
@@ -4,9 +4,373 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars-title">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price card__price--title">Price:</p>
+          <p class="card__price card__price--sum">$2,199</p>
+        </div>
+
+        <div class="card__button-container">
+          <a
+              href="#"
+              class="card__button"
+              data-qa="hover"
+            >
+              BUY
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product image"
+        >
+        <div class="card__description">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </div>
+
+        <div class="card__product-information">
+          <p>Product code: 195434</p>
+        </div>
+
+        <div class="card__stars-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars-title">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price card__price--title">Price:</p>
+          <p class="card__price card__price--sum">$2,199</p>
+        </div>
+
+        <div class="card__button-container">
+          <a
+              href="#"
+              class="card__button"
+              data-qa="hover"
+            >
+              BUY
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product image"
+        >
+        <div class="card__description">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </div>
+
+        <div class="card__product-information">
+          <p>Product code: 195434</p>
+        </div>
+
+        <div class="card__stars-container">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__stars-title">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price card__price--title">Price:</p>
+          <p class="card__price card__price--sum">$2,199</p>
+        </div>
+
+        <div class="card__button-container">
+          <a
+              href="#"
+              class="card__button"
+              data-qa="hover"
+            >
+              BUY
+          </a>
```

Comment:
```suggestion
          <a
              href="#"
              class="card__button"
              data-qa="hover"
          >
              BUY
          </a>
```

Check formatting please